### PR TITLE
Updates for Renku 0.9.0

### PR DESCRIPTION
**Pull request by pameladelgado**
Additionally we will include a new version of contributed project templates.
/deploy

### CODE REVIEW

**Review by ableuler — CHANGES_REQUESTED** [review]
Looking forward to having this release in production. Added some minor suggestions. 

One thing I'm trying to remember is if we're missing some necessary changes to the values files which probably should have been added as PRs got merged. I'm thinking on the necessary ingress annotations for embedding GitLab or the sessions in iframes or instructions on how to change the Keycloak theme, but there might be more.

**Comment by pameladelgado** [comment]
> 
> 
> Looking forward to having this release in production. Added some minor suggestions.
> 
> One thing I'm trying to remember is if we're missing some necessary changes to the values files which probably should have been added as PRs got merged. I'm thinking on the necessary ingress annotations for embedding GitLab or the sessions in iframes or instructions on how to change the Keycloak theme, but there might be more.

Thanks for your feedback. 
Regarding changes in the values file, none should be needed but I've added a section with instructions of configurable items in this version.

**Review by ableuler — DISMISSED** [review]

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,5 +1,54 @@
...
+* New aesthetics, look and feel
+* Sessions (formerly "interactive environments") shown in the UI within their RenkuLab context
+* Issues and Merge Requests shown in the UI within RenkuLab context
+
+Improvements
+~~~~~~~~~~~~
+
+* **Collaboration**: add ``Fork`` tab and ``Open in Tab`` buttons to collaboration pages.
+* **Datasets**: support for dataset marquee image in projects
+
```
Comment (ciyer) [inline CHANGELOG.rst:19]: This looks good. I would add a few more lines here:

```suggestion
* **Sessions**: improve functioning and experience of sessions in iframes
* **File Browser**: allow resizing of file-system navigation view

```

File: CHANGELOG.rst [hunk 2]
```diff
@@ -1,5 +1,54 @@
...
+Bug Fixes
+~~~~~~~~~
+
+* **Projects**: handle primary branches named other than master
+* **Templates**: Template updating issue with  (see this `forum post <https://renku.discourse.group/t/error-during-environment-creation/407/7>`__).
+* **Renku core / CLI**: Update rdflib 6 and remove rdflib-jsonld which could not be installed with setuptools ``>58.0.2``.
+* **CLI**: Fix `renku rm` failure in specific cases.
```
Comment (ciyer) [inline CHANGELOG.rst:26]: User lowercase for consistency:

```suggestion
* **Templates**: template updating issue with  (see this `forum post <https://renku.discourse.group/t/error-during-environment-creation/407/7>`__).
* **Renku core / CLI**: update rdflib 6 and remove rdflib-jsonld which could not be installed with setuptools ``>58.0.2``.
* **CLI**: fix `renku rm` failure in specific cases.
```

**Review by ciyer — CHANGES_REQUESTED** [review]
Excited to be seeing this! 🎉

I made some suggestions for things to add/change in the Changelog.

**Review by ciyer — APPROVED** [review]
🎉

**Pull request merged**